cwms-data-api/src/main/java/cwms/cda/data/dao/JooqDao.java

```diff
@@ -80,11 +80,11 @@ public abstract class JooqDao<T> extends Dao<T> {
     public static final int ORACLE_ECID_MAX_LENGTH = 22;
 
     static ExecuteListener listener = new ExceptionWrappingListener();
-    private static Pattern INVALID_OFFICE_ID = Pattern.compile(
+    private static final Pattern INVALID_OFFICE_ID = Pattern.compile(
         "INVALID_OFFICE_ID: \"([^\"]+)\" is not a valid CWMS office id");
-    private static Pattern VALUE_TOO_LONG = Pattern.compile(
-            "(^ORA-12899: value too large for column )(\".+\"\\.\".+\"\\.\".+\")" +
-                    "( \\(actual: )(\\d+)(, maximum: )(\\d+)(\\)(\\R*)$)");
+    private static final Pattern VALUE_TOO_LONG = Pattern.compile(
+            "^ORA-12899: value too large for column \".+\"\\.\".+\"\\.\"(.+)\" "
```

Comment:
doesn't this only work in Oracle 23.x?

E.g. some of this needs to be optional and account for this lack of data. Or is there already a fall-back I'm not seeing at the moment?